website/docs/reference/project-configs/log-path.md

```diff
@@ -47,12 +47,21 @@ The precedence order is: CLI flag > env var > `dbt_project.yml`
 </VersionBlock>
 
 ## Examples
-### Write logs to a subdirectory named `dbt_logs` instead of `logs`
+### Specify subdirectory using the project config file
```

Comment:
Renaming the heading will affect anyone using an anchor like this:
https://docs.getdbt.com/reference/project-configs/log-path#write-logs-to-a-subdirectory-named-dbt_logs-instead-of-logs

But guessing that won't be a ton of people in this particular case.